test/passing/infix_bind.ml

```diff
@@ -0,0 +1,115 @@
+[@@@ocamlformat "break-infix-before-func=true"]
+
+;;
+f x
+>>= fun y ->
+g y
+>>= fun () ->
+f x >>= fun y -> g y >>= fun () -> f x >>= fun y -> g y >>= fun () -> y ()
```

Comment:
I am not sure to understand why they are all on the same line here. Is that expected? I though the rule was:
- either everything (all the `>>=` sequence) can stay on one line and we are fine
- or we break after every element of the sequence. And we break more the `break-infix-before-func=true` than without.

Reply:
Expected: yes, due to implementation difficulty of forcing boxes to break in sync. I'm not saying it's optimal, just expected.